src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -261,6 +261,12 @@ public partial class AggregateException : System.Exception
     }
     public static partial class AppContext
     {
+        public static string BaseDirectory { [System.Security.SecuritySafeCriticalAttribute]get { throw null; } }
+        public static string TargetFrameworkName { get { throw null; } }
+        [System.Security.SecuritySafeCriticalAttribute]
```

Comment:
SecuritySafeCriticalAttribute annotations are meaningless in reference assemblies. You can remote them.